### DISCUSSION TITLE
why is there no 32 bit support in box64?

### DISCUSSION THREAD

**Discussion by jibbsie** [ptitSeb/box64#225] [General]
I love the project and was trying to get steam working but only works on box86 due to it being both 32/64 bits
I am very curious as to why it there isn't 32 bit support, is it that it is impossible with the hardware or something you are getting to and currently it is early stages make sure everything works?

**Comment by ptitSeb** [comment]
Because of the wrapping libs. 64->32bits wrapping require an extra layer of work, that needs to be done, et complexifie a lot the wrapping.

**Reply by jibbsie** [reply]
oh I think I understand, so they require extra work to make compatible that hasn't been done, if this is the case I understand why you would focus on getting the bare bones then expand out to 32/64 bit integration.
